client/me/connected-application-item/index.jsx

```diff
@@ -153,7 +153,7 @@ class ConnectedApplicationItem extends React.Component {
 				</p>
 
 				{ this.props.translate(
-					'{{detailTitle}}Authorized On{{/detailTitle}}{{detailDescription}}%(date)s{{/detailDescription}}',
+					'{{detailTitle}}Authorized on{{/detailTitle}}{{detailDescription}}%(date)s{{/detailDescription}}',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.